wfe2/wfe_test.go

```diff
@@ -1416,6 +1416,23 @@ func TestNewAccount(t *testing.T) {
 	test.AssertEquals(t, responseWriter.Body.String(), "{\n  \"id\": 1,\n  \"key\": {\n    \"kty\": \"RSA\",\n    \"n\": \"yNWVhtYEKJR21y9xsHV-PD_bYwbXSeNuFal46xYxVfRL5mqha7vttvjB_vc7Xg2RvgCxHPCqoxgMPTzHrZT75LjCwIW2K_klBYN8oYvTwwmeSkAz6ut7ZxPv-nZaT5TJhGk0NT2kh_zSpdriEJ_3vW-mqxYbbBmpvHqsa1_zx9fSuHYctAZJWzxzUZXykbWMWQZpEiE0J4ajj51fInEzVn7VxV-mzfMyboQjujPh7aNJxAWSq4oQEJJDgWwSh9leyoJoPpONHxh5nEE5AjE01FkGICSxjpZsF-w8hOTI3XXohUdu29Se26k2B0PolDSuj0GIQU6-W9TdLXSjBb2SpQ\",\n    \"e\": \"AQAB\"\n  },\n  \"contact\": [\n    \"mailto:person@mail.com\"\n  ],\n  \"agreement\": \"http://example.invalid/terms\",\n  \"initialIp\": \"\",\n  \"createdAt\": \"0001-01-01T00:00:00Z\",\n  \"status\": \"valid\"\n}")
 }
 
+func TestNewAccountWhenAccountHasBeenDeactivated(t *testing.T) {
+	wfe, _ := setupWFE(t)
+	signedURL := fmt.Sprintf("http://localhost%s", newAcctPath)
+	k := loadKey(t, []byte(test3KeyPrivatePEM))
```

Comment:
Could you add a quick note in a comment here that `test3KeyPrivatePEM` is a private key corresponding to a deactivated account in the mock SA's `GetRegistration` test data? I think it makes the rest of the test easier to understand when you know that test account number 3 is deactivated.